cutlass/wmma_matrix.h

```diff
@@ -30,20 +30,20 @@
 #if defined(__CUDACC__) && (!defined(__CUDA_ARCH__) || __CUDA_ARCH__ >= 700)
 #define CUTLASS_USE_WMMA_API
 
-#if defined(__CUDACC__) && (__CUDACC_VER_MAJOR__ >= 10) && (!defined(__CUDA_ARCH__) || __CUDA_ARCH__ >= 720)
+#if defined(__CUDACC__) && (CUDA_VERSION >= 10000) && (!defined(__CUDA_ARCH__) || __CUDA_ARCH__ >= 720)
 #define CUTLASS_USE_INT_WMMA
 #endif
 
-#if defined(__CUDACC__) && (__CUDACC_VER_MAJOR__ >= 10) && (!defined(__CUDA_ARCH__) || __CUDA_ARCH__ >= 750)
+#if defined(__CUDACC__) && (CUDA_VERSION >= 10000) && (!defined(__CUDA_ARCH__) || __CUDA_ARCH__ >= 750)
 #define CUTLASS_USE_SUBBYTE_WMMA
 #endif
 
 #include "stdio.h"
 
-#if __CUDACC_VER_MAJOR__ >= 10
+#if CUDA_VERSION >= 10000
 #include <mma.h>
 #else
-#include <crt/mma.h>
+#include <mma.h>
```

Comment:
I'm not quite sure why 'crt/mma.h' was included originally. The header in CUDA-10.1 has explicit checks to prevent direct inclusion by user.

Should the whole thing be replaced with unconditional `#include <mma.h>` ?

Reply:
We had the same thought, upon review of this. Yes, I agree that `<mma.h>` is the appropriate include.

Reply:
Fixed.